dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/SchedulerServiceImpl.java

```diff
@@ -167,6 +167,13 @@ public Map<String, Object> insertSchedule(User loginUser,
         executorService.checkProcessDefinitionValid(projectCode, processDefinition, processDefineCode,
                 processDefinition.getVersion());
 
+        Schedule scheduleExists =
+                scheduleMapper.queryByProcessDefinitionCode(processDefineCode);
+        if (scheduleExists != null) {
+            throw new ServiceException(Status.SCHEDULE_ALREADY_EXISTS, processDefineCode,
```

Comment:
It's a very good beginning for you to start the first contribution. 
You can prompt the user in a more friendly style, instead of throwing exceptions. Here is an example:
```
log.warn("The schedule already exists.");
putMsg(result, Status.SCHEDULE_ALREADY_EXISTS);
return result;
```